scripts/dev-setup.sh

```diff
@@ -11,11 +11,15 @@ xcrun simctl spawn booted log config --mode "level:off"  --subsystem com.apple.C
 
 # Install Azure CLI with Homebrew for releasing
 # https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-macos?view=azure-cli-latest
-brew update && brew install azure-cli
+brew install azure-cli
```

Comment:
Azure is still used ?

Reply:
Nope, but until releasing 3.9 there is no harm keeping this. I have a more wide cleanup task in my TODO list that includes this ;)